### ISSUE TITLE
Side connectors broken

### ISSUE THREAD

**Issue by ionstorm66** [Eng1N33R/MechanicalTech#24]
~~I can't get power to go through 3 connectors in a row. I want the shaft to be perpendicular to the waterwheel. So I have a UP on the rear of the turbine, a DOWN on top facing the water wheel, and then a SIDE connected to that.~~

~~I get power out of the DOWN if I remove the SIDE.~~

Update: Correction. Side connectors are just broken. Can't get them to work even directly connected to turbines.

**Comment by varencilator** [comment]
I have this same issue, you can use battery base and a cap on top as a work around

**Comment by Bryan123BG** [comment]
Yes I have this issue too! The Angled Connector (Side) actually works, it's just that the textures indicating which side of it transfers energy is wrong; one is correct, while another is on the opposite side of what the texture marks it as.
